File: c/iniciante/1019.c
/* title: Conversão de Tempo
 * number of the problem: 1019
 * date: 18/09/2020
 * author: ltsdw
 */

#include <stdio.h>
#include <math.h>

int main(void) {
    long int s = 0, m = 0, h = 0;
    scanf("%ld", &s);

    if (s >= 60) {
        m = s / 60;
        s = fmod(s, 60);
    }

    if (m >= 60) {
        h = m / 60;
        m = fmod(m, 60);
    }

    printf("%ld:%ld:%ld\n", h, m, s);

    return 0;
}

File: python/iniciante/1019.py
'''
title: Conversão de Tempo
number of the problem: 1019
date: 02/03/2018
author: ltsdw
'''

s = int(input(''))
m = 0
h = 0

if s >= 60:
    m = s / 60
    s %= 60

if m >= 60:
    h = m / 60
    m %= 60

print(f'{h:.0f}:{m:.0f}:{s:.0f}')
